### PR TITLE
Reorganise GUI Layout

### LINKED ISSUE #95: GUI layout to split into 2 sections
- [x] Expand command box at the bottom

### PR DESCRIPTION
**Pull request by saad-haider-pp**
Resolves #95 

Result bar moved to the bottom. Delivery section fills up remaining space.
image

The padding between the quantity and max quantity has been reduced by half as well.

### CODE REVIEW

File: src/main/resources/view/ItemListCard.fxml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.text.Text?>
 <?import javafx.scene.text.TextFlow?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (Wincenttjoi) [inline src/main/resources/view/ItemListCard.fxml]: Is this change to /11.0.1 automated by your intellij system or you changed it manually?

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -33,38 +33,35 @@
...
-              <StackPane fx:id="itemListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+              <StackPane fx:id="itemListPanelPlaceholder" VBox.vgrow="ALWAYS" />
             </VBox>
 
-            <VBox>
-              <children>
-                <StackPane VBox.vgrow="NEVER" fx:id="" styleClass="pane-with-border"
-                           minHeight="295" prefHeight="295">
+               <VBox minHeight="-Infinity" prefHeight="440.0">
+                <StackPane fx:id="deliveryDisplayPlaceholder" minHeight="-Infinity" prefHeight="440.0"
```
Comment (Wincenttjoi) [inline src/main/resources/view/MainWindow.fxml]:
```suggestion
                <StackPane fx:id="deliveryListPanelPlaceholder" minHeight="-Infinity" prefHeight="440.0"
```

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -50,6 +51,9 @@
     @FXML
     private StackPane resultDisplayPlaceholder;
 
+    @FXML
+    private StackPane deliveryDisplayPlaceholder;
```
Comment (Wincenttjoi) [inline src/main/java/seedu/address/ui/MainWindow.java]:
```suggestion
    private StackPane deliveryItemListPlaceholder;
```

Just to keep it more consistent with itemListPanelPlaceholder, what do you think? Just some minor stuff

File: src/main/resources/view/ItemListCard.fxml (hunk 1 above)
Comment (saad-haider-pp) [reply src/main/resources/view/ItemListCard.fxml]: oh didn't realise that happened, must have been automatic

**Comment by Wincenttjoi** [comment]
WIP: as discussed in #95 
- [x] GUI help window toolbar
- [x] Colouring difference (red vs green)

Please tick once done so that we know when to merge 👍

**Comment by saad-haider-pp** [comment]
@Wincenttjoi I've made the changes you requested, plus changed the whole structure of the DeliveryList GUI to match that of the ItemList. (Tonnes of headache cuz debugging GUI is super hard). 

I also wrote the basic code in MainWindow.java for initialising the values into the DeliveryList on starting the app, with currently an empty ObservableList<Delivery> being initialised temporarily. The Delivery class I wrote is also currently just a stub with no functionality.

**Comment by Wincenttjoi** [comment]
> @Wincenttjoi I've made the changes you requested, plus changed the whole structure of the DeliveryList GUI to match that of the ItemList. (Tonnes of headache cuz debugging GUI is super hard).
> 
> I also wrote the basic code in MainWindow.java for initialising the values into the DeliveryList on starting the app, with currently an empty ObservableList being initialised temporarily. The Delivery class I wrote is also currently just a stub with no functionality.

Thanks for changing and doing more than required! LGTM

File: src/main/java/seedu/address/ui/DeliveryCard.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+import javafx.scene.layout.Region;
+import seedu.address.model.delivery.Delivery;
+
+/**
+ * An UI component that displays information of a {@code Item}.
+ */
+public class DeliveryCard extends UiPart<Region> {
+
+    private static final String FXML = "ItemListCard.fxml";
```
Comment (Wincenttjoi) [inline src/main/java/seedu/address/ui/DeliveryCard.java]: Is this meant to be DeliveryListCard.fxml or sth along that line? Or you plan to merge the different cards into same style hence they both refer to ItemListCard?

Comment (saad-haider-pp) [reply src/main/java/seedu/address/ui/DeliveryCard.java]: thanks for catching that! yes it should be DeliveryListCard

Comment (saad-haider-pp) [reply src/main/java/seedu/address/ui/DeliveryCard.java]: have fixed it

**Pull request merged**